administrator-manual/en/nsent/_static/nethesis.css

```diff
@@ -309,4 +343,38 @@ footer .pull-right a:hover {
     .dropdown-menu.globaltoc li:nth-child(4n+0) {
         border-right: none;
     }
+    .wy-nav-content {
+        background-color: white;
+    }
+
 }
+.rst-content .sidebar {
+    border:4px #e1e4e5 !important;
+    }
+.wy-menu-vertical header,
+.wy-menu-vertical p.caption {
+    line-height:21px !important;
+    }
+.wy-side-nav-search>div.version {
+    font-size: 0px !important;
+    display:none;
+    }
+.wy-nav-content-wrap {
+    background:#efefef !important;
+    }
+.wy-side-nav-search input[type="text"] {
+    width: 100% !important;
+    border-radius: 70px !important;
+    padding: 6px 10px !important;
+    border-color: #343131 !important;
+    line-height: 21px !important;
+}
+.rst-content .sidebar .sidebar-title {
+    border-radius: 15px !important;
+   }
+.wy-nav-top {
+    background:#343131 !important;
+}
+.wy-menu-vertical li.current {
+    background:#c9c9c9 !important;
+}
```

Comment:
Text files usually end with a NEWLINE. 

I guess you have to tweak your editor configuration for white space management ;)

Reply:
Thanks for all the review, i'll charge in few moments a new commit with the fix